ring/partition_instance_ring_test.go

```diff
@@ -55,11 +61,14 @@ func TestPartitionInstanceRing_GetReplicationSetsForOperation(t *testing.T) {
 				},
 			},
 			instancesRing: &Desc{Ingesters: map[string]InstanceDesc{
-				"instance-1-a": {Id: "instance-1-a", State: ACTIVE, Timestamp: now.Unix()},
-				"instance-2-a": {Id: "instance-2-a", State: ACTIVE, Timestamp: now.Unix()},
-				"instance-2-b": {Id: "instance-2-b", State: ACTIVE, Timestamp: now.Unix()},
+				"instance-1-a": {Id: "instance-1-a", State: ACTIVE, Zone: "a", Timestamp: now.Unix()},
+				"instance-2-a": {Id: "instance-2-a", State: ACTIVE, Zone: "a", Timestamp: now.Unix()},
+				"instance-2-b": {Id: "instance-2-b", State: ACTIVE, Zone: "b", Timestamp: now.Unix()},
```

Comment:
the reverse `partition-zone` naming is confusing

Reply:
Agree. Fixed in https://github.com/grafana/dskit/pull/489/commits/a2609c759a5226d700fb74d8b806bc955f719464